coral/src/app/features/topics/list/TopicList.tsx

```diff
@@ -1,13 +1,53 @@
 import { Flexbox } from "@aivenio/design-system";
 import { TopicCard } from "src/app/features/topics/list/TopicCard";
 
+const exampleCard = {
```

Comment:
Not all properties are really required. We could only define here the ones we need. Especially when we do not annotate the variable to be `Topic`

Reply:
🤔  I think the comment is on an older commit, right? The `exampleCard` does not exist on the current state of the branch anymore.